docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:8.0
+    container_name: mysql_db
+    environment:
+      MYSQL_ROOT_PASSWORD: root_password
+      MYSQL_DATABASE: todolist_db
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
+    volumes:
+      - mysql_data:/var/lib/mysql
+    ports:
+      - "3306:3306"
+
+  app:
+    build: .
+    container_name: todolist_app
+    depends_on:
+      - db
+    environment:
+      DB_HOST: db
+      DB_PORT: 3306
+      DB_NAME: todolist_db
+      DB_USER: user
+      DB_PASSWORD: password
+    ports:
+      - "8000:8000"
+    volumes:
+      - .:/app
+    entrypoint: ["/entrypoint.sh"]
```

Comment:
Ensure that the `entrypoint.sh` script is correctly set up and executable. This script should handle any necessary initialization tasks for the application.